app/mailers/previews/candidate_mailer_preview.rb

```diff
@@ -43,6 +43,48 @@ def new_referee_request_with_email_bounced
     CandidateMailer.new_referee_request(application_form, reference, reason: :email_bounced)
   end
 
+  def chase_candidate_decision_with_one_offer
+    application_form = OpenStruct.new(
+      candidate: OpenStruct.new(email_address: 'candidate@email.com'),
+      first_name: 'Ting',
+    )
+    application_choice = OpenStruct.new(
+      provider: OpenStruct.new(name: 'Provider Name'),
+      decline_by_default_at: Time.zone.now,
+      application_form: application_form,
+      course: OpenStruct.new(name: 'Course Name'),
+    )
```

Comment:
Is there a reason not to use FactoryBot like the other previews?

Reply:
Yes, `build` does not allow me to create application choices with courses and providers unless I use `create`, which would pollute the db

Reply:
A lot of the previews are using `create`. I think the best solution here is to have a transaction rollback mechanism like the one Jake tried to implement a couple of weeks ago. I'd like to try and get that working.